src/libraries/System.Text.Json/gen/JsonConstants.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Text.Json
+{
+    internal static partial class JsonConstants
+    {
+        public const string GlobalNamespaceValue = "<global namespace>";
```

Comment:
Is this defined by Roslyn? (do they have a constant somewhere?)

Reply:
It's used in a few places in their repo, e.g. [here](https://github.com/dotnet/roslyn/blob/93189a1f36b62ded79375228027c385f7911e74e/src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.cs#L156), but not exposed publicly AFAIK .